jsapp/js/components/accountSettings.es6

```diff
@@ -143,7 +144,7 @@ export var AccountSettings = React.createClass({
     var accountName = stores.session.currentAccount.username;
 
     return (
-      <DocumentTitle title={`${accountName} | KoboToolbox`}>
+      <DocumentTitle title={`${accountName} | ${ONA_TITLE}`}>
```

Comment:
I don't think I have changed the variable style. Check out [this line](https://github.com/onaio/kpi/pull/5/files#diff-2420384d1f3d98c06669a161ffbcd860R8) just before my definition of `ONA_TITLE`. I was trying to be consistent with what already exists. And it is a constant, I don't think it will change any time soon